src/components/Project/Project.scss

```diff
@@ -0,0 +1,115 @@
+@import '../../variables.scss';
+
+.project-page {
+  padding: 0% 3%;
+  background: {
+    attachment: fixed;
+    color: rgba(0,0,0,0.5);
+    image: url('../../images/wave3.png');
+    position: center;
+    repeat: no-repeat;
+    size: cover;
+  }
+  background-blend-mode: darken;
+  margin: 0;
+  min-height: 100%;
+  min-width: 100%;
+  overflow-x: scroll;
+}
+
+.project-nav {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 20px;
+}
+
+.nav-buttons {
+  display: flex;
+  width: 15%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.palettes-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.proj-palette {
+  background-color: black;
+  display: flex;
+  width: 48%;
+  justify-content: space-around;
+  align-items: center;
+  padding: 10px;
+  margin-bottom: 3%;
+}
+
+.proj-palette-name {
+  font-weight: normal;
+  font-size: 2em;
+}
+
+.trash-icon {
+  height: 45px;
+  cursor:  pointer;
+  &:hover,
+  &:focus {
+    transform: scale(1.1);
+  }
+}
+
+.back-arrow-icon {
+  transform: rotate(90deg);
+  height: 27px;
+  cursor: pointer;
+  margin-right: -26px;
+  color: $light-text-color;
+}
+
+.back-button {
+  color: $light-text-color;
+  font-size: 1.2rem;
+  margin: {
+    bottom: auto;
+    top: auto;
+  }
+  &:hover {
+    color: $medium-text-color;
+    text-decoration: none;
+  }
+}
+
+.project-div {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 10% 5%;
+}
+
+.delete-project-button {
+  background-color: #a63232;
+  border: none;
+  font-size: 1.2em;
+  color: white;
+  padding: 0.5% 2%;
+  border-radius: 15px;
+  &:hover,
+  &:focus {
+    transform: scale(1.1);
+  }
+}
+
+.color-box-container {
```

Comment:
Can you try applying `margin-left: auto;` to this? I think that should make it so that these are all aligned against the delete button.

Reply:
Good call - I noticed they weren't aligned but forgot about it by the time i pushed up!  Margin-left auto fixed it :)